src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -203,6 +248,30 @@ private static String getTextOrEmptyString(TextField textField) {
     return textField.getText() != null ? textField.getText().trim() : "";
   }
 
+  private static String convertConfigWinnerElectionModeToGuiText(ContestConfig config) {
```

Comment:
I have a general concern about this approach. Up until this point we've been successful at keeping a tight alignment between the raw config file and the UI, which I think has been really helpful for the maintainability of our code. In this PR you're introducing various pieces of tricky logic to map from the config to the UI and vice versa because they don't represent information in the same way. If we want to make these UI changes, why don't we just update the config file schema accordingly? We can add simple migration logic to convert from an old file to a current one in order to preserve backward compatibility. This might be a bit more work in the short term, but I think it will save us pain in the long run.

Reply:
Agree, I wish we could keep it 1:1, but it just doesn't seem feasible unless we revamp not only the config file, but the logic that parses it. This new list of winner election modes both includes new entries (which are based not only on `winnerElectionMode` but also how other parameters are set), and gets rid of an entry. Please [review this comment](https://github.com/BrightSpots/rcv/issues/461#issuecomment-668365720) and let me know how we could update the config file schema without making serious changes to the other back-end logic (changes that I'd argue are better for a major version increase: 2.0.0, rather than 1.2.0.

Reply:
Done!